Automattic/wp-calypso
# Gutenberg: Use alt Twitter icon for Publicize

This PR updates the Twitter icon that we use for our Publicize extension, as requested by @MichaelArestad in https://github.com/Automattic/wp-calypso/pull/29317#issuecomment-446353031

#### Changes proposed in this Pull Request

* Use a more appropriate icon for the Twitter service in our Publicize extension for Gutenberg

#### Preview

Before:
![](https://cldup.com/kZdjTyimMz.png)

After:
![](https://cldup.com/0s_PWpSvqS.png)


#### Testing instructions

* Spin up calypso.live from the link below.
* Select a simple WP.com site.
* Start a post.
* Click the Jetpack icon.
* Connect a Twitter account to your site.
* Verify the new icon appears.
